en/core-libraries/email.rst

```diff
@@ -483,7 +483,7 @@ following::
         {
             $this
                 ->to($user->email)
-                ->subject(sprintf('Welcome %s', $user->name))
+                ->subject(__('Welcome %s', $user->name))
```

Comment:
http://book.cakephp.org/3.0/en/core-libraries/global-constants-and-functions.html#global-functions

> . Strings used for translations are treated as format strings for sprintf().

http://book.cakephp.org/3.0/en/core-libraries/internationalization-and-localization.html#using-different-formatters

Probably the whole documentation needs an update... :confused: